test/test_utils/security.py

```diff
@@ -1,13 +1,133 @@
+from abc import abstractmethod
```

Comment:
@saimidu to complete review of test/test_utils/security.py. This file is not reviewed yet.